asdf/resource.py

```diff
@@ -25,6 +26,61 @@
 ]
 
 
+class ResourceMappingProxy(Mapping):
```

Comment:
There's less of a strong case for this wrapper, but since the plugin can be a simple dict, it's not obvious what extension package provided them.  This'll give us some clues when we're trying to figure out where a busted resource plugin is coming from.